lib/Cargo.toml

```diff
@@ -106,6 +106,7 @@ storekey = "0.5.0"
 thiserror = "1.0.40"
 tikv = { version = "0.1.0", package = "tikv-client", optional = true }
 tikv-client-proto = { version = "0.1.0", optional = true }
+time = { version = "0.3.20" }
```

Comment:
I don't see why we need a `time` dependency, considering `chrono` aims to be "a feature-complete superset of `time`"

Reply:
Will remove thanks